site/pages/create-disruption/__snapshots__/create-disruption.test.tsx.snap

```diff
@@ -1017,6 +1017,7 @@ exports[`pages > CreateDisruption > should render correctly with inputs 1`] = `
               </div>
             </fieldset>
           </div>
+          <legend />
```

Comment:
These snapshots all seem to have empty legends - is that expected? Should we be including the new text in the snapshots?

Reply:
done